process_pub_database/get_data_cpdl.py

```diff
@@ -8,32 +8,108 @@
 import time
 import unicodedata
 
+from utils import *
```

Comment:
it's better to be explicit here, I'd do `import utils` and then `utils.whatever` when you need to use it